.github/workflows/unit-tests.yml

```diff
@@ -45,7 +48,7 @@ jobs:
       - name: Run unit tests
         if: inputs.os == 'ubuntu-latest' && inputs.python-version != '3.10'
         run: make test
-      - name: Run unit tests without spark sequentially (Windows)
+      - name: Run unit tests without spark sequentially
```

Comment:
Might still be useful to have `(Windows)` in the name here, or does that not show on the build?